src/components/docs/components/SampleDataProps.tsx

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react'
+import { SampleConfig } from '../../../sample-data'
+
+/**
+ * This file is to reuse JSDoc in MDX docs.
+ */
```

Comment:
@alexbchr Not quite sure I understand what this is for. 🕵️

Reply:
Well... I agree that this is maybe not the more straightforward thing ever, but since `useSampleData` is a hook (and not a React Component), we can't show the Prop Table for it in Storybook (with JSDoc and everything). However, by creating a Dummy Component taking the same props as `useSampleData`, we are able to show the Prop Table generated with the JSDoc.

This is only used in the MDX docs (Storybook).